data/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# org.ayatana.common.gschema.xml
+
+find_package(GSettings)
+set(ENV{LC_ALL} "C")
+execute_process(COMMAND intltool-merge -quiet --xml-style --utf8 --no-translations "${CMAKE_CURRENT_SOURCE_DIR}/org.ayatana.common.gschema.xml.in" "${CMAKE_CURRENT_BINARY_DIR}/org.ayatana.common.gschema.xml")
+add_schema("org.ayatana.common.gschema.xml")
```

Comment:
This commit leaves the project in a broken state. Please commit `org.ayatana.common.gschema.xml` with it.